src/model/Subscription.model.js

```diff
@@ -17,7 +17,7 @@
 
 import { toRaw } from 'vue'
 import ViewState from '@/model/ViewState.model'
-import Alert from '@/model/Alert.model'
+import { Alert } from '@/model/Alert.model'
```

Comment:
Non-functional change. For some reason the type recognition in VSCode wasn't working for a default export. So that's why I've changed it